Explorer/Assets/Scripts/ECS/StreamableLoading/DeferredLoading/DeferredLoadingSystem.cs

```diff
@@ -0,0 +1,91 @@
+using Arch.Core;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using ECS.Abstract;
+using ECS.Prioritization.Components;
+using ECS.StreamableLoading.AssetBundles;
+using ECS.StreamableLoading.AssetBundles.Manifest;
+using ECS.StreamableLoading.Common.Components;
+using ECS.StreamableLoading.Textures;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Unity.Collections.LowLevel.Unsafe;
+using UnityEngine;
+
+namespace ECS.Prioritization.DeferredLoading
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [UpdateAfter(typeof(PrepareAssetBundleManifestParametersSystem))]
+    [UpdateBefore(typeof(LoadAssetBundleManifestSystem))]
+    [UpdateBefore(typeof(LoadTextureSystem))]
+    [UpdateBefore(typeof(LoadAssetBundleSystem))]
+    public partial class DeferredLoadingSystem<TAsset, TIntention> : BaseUnityLoopSystem where TIntention: struct, ILoadingIntention
```

Comment:
I think it will be a nightmare to track all the implementations by generic arguments and put all `UpdateBefore` attributes to the same class. Please consider introducing individual sub-classes for each type and annotating them with the required attributes only.